client/ayon_maya/plugins/publish/validate_animation_product_type_publish.py

```diff
@@ -0,0 +1,42 @@
+import ayon_maya.api.action
+from ayon_core.pipeline.publish import (
+    PublishValidationError,
+    ValidateContentsOrder,
+)
+from ayon_maya.api import plugin
+
+
+class ValidateAnimationProductTypePublish(plugin.MayaInstancePlugin):
```

Comment:
If this is a validation that anyone wants - it should be studio specific and configurable in settings. There are valid reasons to allow exporting multiple product types at once.

Reply:
Sorry, my bad... I thought this was validating whether there were more than one product types being exports BUT it doesn't. It validates whether at least one product type is being exported - which is a reasonable validation.